iml-agent/src/action_plugins/stratagem/action_cloudsync.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+extern crate futures;
+extern crate tokio;
+
+use crate::agent_error::{ImlAgentError, RequiredError};
+use crate::lustre::search_rootpath;
```

Comment:
```suggestion
use crate::{
    agent_error::{ImlAgentError, RequiredError},
    lustre::search_rootpath,
};
```

Compress to a single import.

Reply:
Done